experiments/copter.parm

```diff
@@ -0,0 +1,104 @@
+EK2_ENABLE      1
+FRAME_TYPE	0
+MAG_ENABLE	1
+FS_THR_ENABLE   1
+BATT_MONITOR    4
+CH7_OPT         7
+COMPASS_LEARN   0
+COMPASS_OFS_X   5
+COMPASS_OFS_Y   13
+COMPASS_OFS_Z   -18
+COMPASS_OFS2_X   5
+COMPASS_OFS2_Y   13
+COMPASS_OFS2_Z   -18
+FENCE_RADIUS    150
+FRAME_CLASS     1
+RC1_MAX         2000.000000
+RC1_MIN         1000.000000
+RC1_TRIM        1500.000000
+RC2_MAX         2000.000000
+RC2_MIN         1000.000000
+RC2_TRIM        1500.000000
+RC3_MAX         2000.000000
+RC3_MIN         1000.000000
+RC3_TRIM        1500.000000
+RC4_MAX         2000.000000
+RC4_MIN         1000.000000
+RC4_TRIM        1500.000000
+RC5_MAX         2000.000000
+RC5_MIN         1000.000000
+RC5_TRIM        1500.000000
+RC6_MAX         2000.000000
+RC6_MIN         1000.000000
+RC6_TRIM        1500.000000
+RC7_MAX         2000.000000
+RC7_MIN         1000.000000
+RC7_TRIM        1500.000000
+RC8_MAX         2000.000000
+RC8_MIN         1000.000000
+RC8_TRIM        1500.000000
+FLTMODE1        7
+FLTMODE2        9
+FLTMODE3        6
+FLTMODE4        3
+FLTMODE5        5
+FLTMODE6		0
+SUPER_SIMPLE	0
+SIM_GPS_DELAY   1
+SIM_ACC_RND     0
+SIM_GYR_RND     0
+SIM_WIND_SPD    0
+SIM_WIND_TURB   0
+SIM_BARO_RND    0
+SIM_MAG_RND     0
+SIM_GPS_GLITCH_X    0
+SIM_GPS_GLITCH_Y    0
+SIM_GPS_GLITCH_Z    0
+# we need small INS_ACC offsets so INS is recognised as being calibrated
+INS_ACCOFFS_X   0.001
+INS_ACCOFFS_Y   0.001
+INS_ACCOFFS_Z   0.001
+INS_ACCSCAL_X   1.001
+INS_ACCSCAL_Y   1.001
+INS_ACCSCAL_Z   1.001
+INS_ACC2OFFS_X   0.001
+INS_ACC2OFFS_Y   0.001
+INS_ACC2OFFS_Z   0.001
+INS_ACC2SCAL_X   1.001
+INS_ACC2SCAL_Y   1.001
+INS_ACC2SCAL_Z   1.001
+INS_ACC3OFFS_X   0.000
+INS_ACC3OFFS_Y   0.000
+INS_ACC3OFFS_Z   0.000
+INS_ACC3SCAL_X   1.000
+INS_ACC3SCAL_Y   1.000
+INS_ACC3SCAL_Z   1.000
+MOT_THST_EXPO 0.5
+MOT_THST_HOVER  0.36
+# flightmodes
+# switch 1 Circle
+# switch 2 LAND
+# switch 3 RTL
+# switch 4 Auto
+# switch 5 Loiter
+# switch 6 Stab
+# STABILIZE 	0 !
+# ACRO 			1
+# ALT_HOLD 		2
+# AUTO 			3 !
+# GUIDED 		4
+# LOITER 		5 !
+# RTL 			6 !
+# CIRCLE 		7 !
+# POSITION 		8
+# LAND 			9 !
+# *********************************
+WPNAV_RADIUS    20
```

Comment:
This is quite a large value. Ideally, we want this to be as close as possible to zero. See http://ardupilot.org/copter/docs/parameters.html#wpnav-parameters

Reply:
The minimum for this parameter is 10cm. So I thought 20cm would be a reasonable number. I can set it to 10 though.

Reply:
Ah. I thought that the parameter was measured in metres. 20cm seems reasonable.